src/test/java/org/mondemand/tests/ClientTest.java

```diff
@@ -1008,19 +1008,22 @@ public void testIncrementMap()
       client.increment(contexts, "key1");
       client.increment(contexts, "key2", 100l);
     }
+    ContextList contextsCopy = new ContextList();
+    contextsCopy.addContext(context);
```

Comment:
I'd suggest creating a new equivalent Context here as well.  As this stands, I think you are not covering much of Context.equals() with this test